api/config/module/chaos/experimentation/xds/v1/xds.proto

```diff
@@ -26,4 +26,10 @@ message Config {
 
   // The interval at which to send heartbeat responses for TTL'd resources.
   google.protobuf.Duration heartbeat_interval = 6;
+
+  // list of clusters for which ECDS is enabled. * means it will be enabled for all
+  repeated string ecds_enabled_clusters = 7;
+
+  // A prefix to use to override runtime keys for ECDS
+  string ecds_fault_runtime_prefix = 8;
```

Comment:
I'm not sure if its worth it making this configurable (just hard code to something nobody would ever configure?), but if are we should speak to its purpose a bit more